### PR TITLE
Backport viewkeeper fix and release 4.9.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.9.1 (2021-12-03)
+
+- Fixed the viewkeeper direct to ignore the viewport margin when attaching to a scrollable host. [#279](https://github.com/blackbaud/skyux-core/pull/279)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:3]: Not sure what direct is referring to. Is it a function? If so, should the formatting be different?

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Should we just delete `direct`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 4.9.1 (2021-12-03)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.9.1 (2021-12-06)
```

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That was a typo. It should have said `directive`. I’ve updated it.

**Review by blackbaud-johnly — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**